src/supermarket/app/controllers/api/v1/cookbook_uploads_controller.rb

```diff
@@ -44,7 +44,7 @@ def create
     rescue
       render_not_authorized([t('api.error_messages.unauthorized_upload_error')])
     else
-      cookbook_upload.finish do |errors, cookbook, cookbook_version|
+      cookbook_upload.finish(current_user) do |errors, cookbook, cookbook_version|
```

Comment:
`#finish` no longer needs `current_user` passed in since `user` gets passed in and set in `CookbookUpload.new`